server_action_mass_edit/models/ir_actions_server_mass_edit_line.py

```diff
@@ -47,6 +47,7 @@ class IrActionsServerMassEditLine(models.Model):
         default=False,
         help="Apply default domain related to field",
     )
+    apply_onchanges = fields.Boolean(help="Play field onchanges before writing value")
```

Comment:
Maybe we can add more information in the help text. Perfomance possible problem if it is executed on a lot of records versus inconsistency data if onchange are not executed.

Note : for new version (v18), I think we can put default="True", as it is the most expected behaviour.